code/game/objects/items.dm

```diff
@@ -673,6 +673,8 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	return 0
 
 /obj/item/attack_animal(mob/living/simple_animal/M)
+	if (src.item_flags & CAN_BE_HIT)

```

Comment:
`obj_flags` instead of `src.item_flags`. I know said item_flags in IRC earlier but i was having a brainfart moment